golang/pkg/crane/k8s/deployment.go

```diff
@@ -276,13 +276,18 @@ func (d *Deployment) GetPodDeployment(namespace, name string) (*kappsv1.Deployme
 	return client.AppsV1().Deployments(namespace).Get(d.ctx, owner.Name, metaV1.GetOptions{})
 }
 
+func getReplicaCount(count uint8) int32 {
```

Comment:
The name doesn’t indicate what the function does. Ideas: `ensureReplicaCount`, `atLeastOneReplica`, `replicaCountOrDefault`.